ospray_modules/optix/framebuffer/FrameBuffer.cpp

```diff
@@ -117,14 +117,15 @@ float FrameBuffer::endFrame(const float /*errorThreshold*/)
 const void* FrameBuffer::mapDepthBuffer()
 {
     if (colorBuffer)
-        depthBuffer = (float*)colorBuffer;
+        depthBuffer = colorBuffer;
     return (const void*)depthBuffer;
```

Comment:
I cannot judge this code without studying the whole design, but this looks strange.

Reply:
I am not sure either but I think  it is like this to hide the fact that we do not provide a depth buffer from OptiX.

Reply:
That looks totally wrong, but I guess it can go in another PR